build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+mix deps.get
+
+mix ecto.create
+mix ecto.migrate
+
+mix test
+
+mix phx.server
```

Comment:
This is a simple build script which is fine :)
In the future we should consider using `mix release` instead 🙇